src/components/MentionList.vue

```diff
@@ -0,0 +1,111 @@
+<template>
+  <div class="items">
+    <template v-if="items.length">
+      <button
+        class="item"
+        :class="{ 'is-selected': index === selectedIndex }"
+        v-for="(item, index) in items"
+        :key="index"
+        :value="item.gid"
+        @click="selectItem(index)"
+      >
+        {{ item.name }}
+      </button>
+    </template>
+    <div class="item" v-else>
+      No result
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import { User } from "@/types/asana";
+import { defineComponent, PropType } from "vue";
+
+export default defineComponent({
+  props: {
+    items: {
+      type: Array as PropType<User[]>,
+      required: true,
+    },
+    command: {
+      type: Function,
+      required: true,
+    },
+  },
+  data() {
```

Comment:
This is the older vue2 class-style definition. data, watch, methods should all be in setup()